src/components/TitleComponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+
+function RenderTitle({title}) {
+    return (
+        <div class="intro">
+            <h2 class="text-center">{title.name}</h2>
```

Comment:
let's center this using scss instead of bootstrap classes.

Lets just use bootstrap classes for specific things, like navbars.

If it's something that can be done easily we'll write our own styles